frontend/src/bundles/common/services/toast/toast.ts

```diff
@@ -0,0 +1,12 @@
+import { createStandaloneToast } from '@chakra-ui/react';
+
+import { theme } from '~/framework/theme/theme.js';
+
+import { ToastService } from './toast.service.js';
+
+const { toast } = createStandaloneToast({ theme: theme });
+
+const toastService = new ToastService({ toast });
+
+export { ToastService } from './toast.service.js';
+export { toastService };
```

Comment:
Is it possible to combine these 2 exports?

Reply:
if I try to do so, I receive the next error:

Use `export…from` to re-export `NotificationService`.eslint [unicorn/prefer-export-from](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v50.0.1/docs/rules/prefer-export-from.md)